frontend/src/components/shared/ImpactReportField.js

```diff
@@ -9,13 +9,30 @@ import {fetchHowMuchImpacts} from "../../api/howMuchImpactApi";
 import {fetchImpactRisks} from "../../api/impactRiskApi";
 import Dropdown from "./fields/MultiSelectField";
 import {fetchDataTypeInterfaces} from "../../api/generalAPI";
-
+import {isFieldRequired, validateField, validateURI, validateFieldAndURI} from "../../helpers";
+import {fullLevelConfig} from "../../helpers/attributeConfig";
 
 const filterOptions = createFilterOptions({
   ignoreAccents: false,
   matchFrom: 'start'
 });
 
+const attribute2Compass = {
```

Comment:
Please make attribute2Compass a property of ImpactReportField, instead of redefining it again here. You may want to import it directly from AddEditImpactReport. Please look at CounterFactualField to get an idea.